L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -151,9 +151,9 @@
 from L1Trigger.L1TTrackMatch.l1tTrackerEtMiss_cfi import *
 from L1Trigger.L1TTrackMatch.l1tTrackerHTMiss_cfi import *
 # make the input tags consistent with the choice L1VertexFinder above
-l1tTrackJets.L1PVertexCollection  = ("l1tVertexFinder", "l1vertices")
+l1tTrackJets.VertexInputTag  = ("l1tVertexFinder", "l1vertices")
 l1tTrackFastJets.L1PrimaryVertexTag  = ("l1tVertexFinder", "l1vertices")
-l1tTrackJetsExtended.L1PVertexCollection  = ("l1tVertexFinder", "l1vertices")
+l1tTrackJetsExtended.VertexInputTag  = ("l1tVertexFinder", "l1vertices")
```

Comment:
Should these be `InputTag`s?

`l1tTrackJetsExtended.VertexInputTag` is cloned from `l1tTrackJets` which has `VertexInputTag` as a `cms.InputTag`:

https://github.com/cms-sw/cmssw/blob/e0e96bc60bd0e165e676542c720fa8dcd873f9ad/L1Trigger/L1TTrackMatch/python/l1tTrackJets_cfi.py#L4-L6

Reply:
Hello,

Yes thus is the problem. I changed the parameter names but i was not sure if i need to change the input tags themselves. I did it now. thanks